### PR TITLE
feat: verbose flag

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
## Description
This PR adds optional `printDocs` flag that prints overview of indexed documentation chunks. This can be useful for debugging.

Output looks like this:

Snímek obrazovky 2023-09-21 v 12 14 43

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by kysely — COMMENTED** [review]
I feel like this flag is too command-specific, yet doesn't feel like its feature. Perhaps some more generic flag that'd also work for other commands eventually would be a better fit? I'm thinking `--verbose` `--debug` etc

**Comment by Jakub-Vacek** [comment]
> I feel like this flag is too command-specific, yet doesn't feel like its feature. Perhaps some more generic flag that'd also work for other commands eventually would be a better fit? I'm thinking `--verbose` `--debug` etc

Good point, renamed to `verbose`

File: src/commands/prepare.ts [hunk 1]
```diff
@@ -119,19 +127,34 @@ This command prepares a Provider JSON metadata definition that can be used to ge
...
       },
       { userError, ux }
     );
 
-    const providerJsonPath = await writeProviderJson(providerJson, {
-      logger,
-      userError,
-    });
+    const docs = providerJsonResult.docs
+      ? `\n{bold gray Idexed documentation:}\n${providerJsonResult.docs
```
Comment (janhalama) [inline src/commands/prepare.ts:144]: issue: looks like the `gray` formating option do not work:
image

Comment (janhalama) [inline src/commands/prepare.ts:144]: issue: typo

```suggestion
      ? `\n{bold gray Indexed documentation:}\n${providerJsonResult.docs
```

**Review by janhalama — COMMENTED** [review]
Looks good, I found some problems with the formatting of the text output.

**Comment by Jakub-Vacek** [comment]
Fixed formatting:
Snímek obrazovky 2023-10-03 v 12 24 23

**Comment by kysely** [comment]
> Fixed formatting:
> Snímek obrazovky 2023-10-03 v 12 24 23

In the picture, was the command called with `--verbose`?

**Comment by Jakub-Vacek** [comment]
Oh, yes. It was called with `-v`

**Review by kysely — APPROVED** [review]
Looks fine from me

**Pull request merged**